seed/lib/mcm/mapper.py

```diff
@@ -99,11 +99,21 @@ def apply_column_value(raw_column_name, column_value, model, mapping, is_extra_d
     :rtype: model inst
     """
 
-    def is_pint_column(column_name):
+    def is_quantity_type_column(column_name):
         """Test if the column_name is a QuantityField"""
-        # TODO rgm change this to a test against a set (the 7 db column names) after feature release
         # TODO rgm re-locate to someplace else, eg. Column
-        return re.search(r'_pint$', column_name)
+        quantity_column_names = [
```

Comment:
this is good for now. I think we should eventually put this in the mapping jsons as a field and then do a simple lookup on that.